joomla/joomla-cms
# Transform language flags into buttons in default pages/templates

#### Description

This PR takes the contributions from https://github.com/joomla/joomla-cms/pull/8815 and transforms the language flags in com_menus and com_templates into buttons with tooltips to be sincronized with the set/unset default buttons.

Example:
![image](https://cloud.githubusercontent.com/assets/9630530/12076517/cbb581b4-b1a5-11e5-8225-a7c53fa5d508.png)
#### How to test
1. Fresh Joomla install with multilanguage (several languages).
2. Apply this patch
3. Go to menus and select/deselect default pages (global and language).
4. Go to templates and select/deselect default templates (global and language).
5. For both, check the buttons are ok and working properly (and their tooltips too)
